proto/interchainqueries/genesis.proto

```diff
@@ -36,6 +36,9 @@ message RegisteredQuery {
 
   // The remote chain last block height when the query result was updated.
   uint64 last_submitted_result_remote_height = 10;
+
+  // Amount of coins deposited for the query.
+  uint64 deposit = 11;
```

Comment:
You added a new field into `RegisteredQuery` struct. Please, do not forget to update wasm bindings for relevant structure (in [Neutron itself](https://github.com/neutron-org/neutron/blob/caf377b38db1723f2d3941e61f450af5830de212/wasmbinding/bindings/query.go#L53) and in the [Neutron CosmWasm SDK](https://github.com/neutron-org/neutron-contracts/blob/a47bfac69667da57f8bf6ea81c9f16240e145c6d/packages/neutron-sdk/src/bindings/types.rs#L25))